tests/Benchmarks/Microsoft.Experimental.Collections/MultiValueDictionaryPerformanceTests.cs

```diff
@@ -0,0 +1,176 @@
+                new MultiValueDictionary<int, string>(); new MultiValueDictionary<int, string>(); new MultiValueDictionary<int, string>();
+            }
+        }
+
+        [Benchmark]
+        [Arguments(0)]
+        [Arguments(1024)]
+        [Arguments(4096)]
+        [Arguments(16384)]
+        public void Ctor_Size(int size)
+        {
+            for (int i = 0; i <= 500; i++)
+            {
+                new MultiValueDictionary<int, string>(size); new MultiValueDictionary<int, string>(size); new MultiValueDictionary<int, string>(size);
+                new MultiValueDictionary<int, string>(size); new MultiValueDictionary<int, string>(size); new MultiValueDictionary<int, string>(size);
+                new MultiValueDictionary<int, string>(size); new MultiValueDictionary<int, string>(size); new MultiValueDictionary<int, string>(size);
+            }
+        }
+
+        [Benchmark]
+        public void GetItem()
+        {
+            IReadOnlyCollection<int> retrieved;
+            for (int i = 0; i <= 10000; i++)
+            {
+                retrieved = dict[1]; retrieved = dict[2]; retrieved = dict[3];
+                retrieved = dict[4]; retrieved = dict[5]; retrieved = dict[6];
+                retrieved = dict[7]; retrieved = dict[8]; retrieved = dict[9];
+            }
+        }
+
+        [Benchmark]
+        public void GetKeys()
+        {
+            IEnumerable<int> result;
+            for (int i = 0; i <= 20000; i++)
+            {
+                result = dict.Keys; result = dict.Keys; result = dict.Keys;
+                result = dict.Keys; result = dict.Keys; result = dict.Keys;
+                result = dict.Keys; result = dict.Keys; result = dict.Keys;
+            }
+        }
+
+        [Benchmark]
+        public void TryGetValue()
+        {
+            for (int i = 0; i <= 1000; i++)
+            {
+                dict.TryGetValue(RandomKey, out var _); dict.TryGetValue(RandomKey, out var _);
+                dict.TryGetValue(RandomKey, out var _); dict.TryGetValue(RandomKey, out var _);
+                dict.TryGetValue(RandomKey, out var _); dict.TryGetValue(RandomKey, out var _);
+                dict.TryGetValue(RandomKey, out var _); dict.TryGetValue(RandomKey, out var _);
+            }
+        }
+
+        [Benchmark]
+        public void ContainsKey()
+        {
+            for (int i = 0; i <= 10000; i++)
+            {
+                dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey);
+                dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey);
+                dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey);
+                dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey); dict.ContainsKey(RandomKey);
+            }
+        }
+
+        [Benchmark]
+        public void ContainsValue()
+        {
+            for (int i = 0; i <= 20000; i++)
```

Comment:
Is it necessary to have such a for loop (along with loop unrolling)? Also, with different iteration counts, how do we reason about/compare the perf results?

cc @adamsitnik